dspace/config/modules/curate.cfg

```diff
@@ -52,3 +52,6 @@ curate.ui.statusmessages = \
      2 = Skip, \
      other = Invalid Status
 
+
+rest.properties.exposed = plugin.named.org.dspace.curate.CurationTask
```

Comment:
This should have a comment to explain it.  Something like:
`# Ensure list of Curation Tasks (defined above) is available via the REST API /api/config/properties endpoint`

Reply:
@tdonohue 

* **Email parameter in UI**
    * I checked this with Kevin, and he provided me with the information regarding the work about this being fixed for the  metadata-import. I'll schedule to have this updated for this piece of code as well.
    * Long story short is that there is a difference between the actual CLI and REST. This will not be handled in general in the scripts and processes framework since this needs to be handled on a par script basis (Since there are scripts that won't have a difference in CLI/REST options)
* With regard to your comment about the **REST API not returning the Curation task output**
    * We are looking into this under the general 'Scripts and Processes' work so this can be handled over the entire codebase and not simply on a per script implementation basis. 
    * That would add in the option to keep the output separately for the scripts and processes.
    * This would also allow us to have more fine-grained tests for these types of functionality. For example checking that the output of the noop curation task contains the correct information)
* **curate.cfg configuration updates**
    * I've commented out `vscan` and `translate` and removed the `curate.ui.*` properties  the configuration as per your request (As your rightfully mentioned `curate.ui.*` isn't used anymore, and no actual tests are related to these properties) 
    * The configuration for `rest.properties.exposed` is related to  https://github.com/DSpace/DSpace/pull/2762. To shield the configuration of the properties, the information that can be retrieved through rest need to be configured as such. I'd advise to keep it, so at least there is an option to retrieve the curation tasks that are enabled. (Not having this config exposed shields this config)
        * Calling /api/config/properties/plugin.named.org.dspace.curate.CurationTask would then return the actual tasks that ARE configured. (So the 4 remaining 'enabled' tasks configured in [the config]( https://github.com/atmire/DSpace/blob/w2p-71513_Curation-tasks-scripts-and-processes/dspace/config/modules/curate.cfg#L11) )

Reply:
@jonas-atmire : Thanks for the summary/response.  A few things to be aware of:

* **Email Parameter in UI / REST**:  Please don't start any work on this yet.  We (the DSpace 7 team) will be discussing this security flaw in next week's meeting to provide a route forward. https://wiki.lyrasis.org/display/DSPACE/2020-07-30+DSpace+7+Working+Group+Meeting  
* **REST API not returning the Curation task output**: Your plan sounds good overall.
* **curate.cfg configuration updates**:  These changes sound good overall...After reviewing the corresponding Angular PR: https://github.com/DSpace/dspace-angular/pull/799 I found there definitely *is* a need for the `/api/config/properties/plugin.named.org.dspace.curate.CurationTask` to be exposed.  (Currently, *both* PRs are configuring a list of "enabled" curation tasks.   Rather than hardcoding this list *both* on the frontend and backend, ideally the frontend should check the configuration of the backend.)    So, you are correct, this is needed & the changes look good.

Reply:
@tdonohue, just a bit of cross referencing
**REST API not returning the Curation task output:**
This is currently being fixed in the following PR: 
https://github.com/DSpace/DSpace/pull/2905